src/components/write_to_hub_controlnet/fondant_component.yaml

```diff
@@ -1,22 +1,14 @@
 name: Write to hub
```

Comment:
Do we still need the custom component spec? Couldn't we just use the new interface like this: 
```python
segmentations.apply(
       "write_to_hub",
        arguments={
            ...
            }
        },
       produces={
       "image": pa.binary(),
       "image_width": pa.int32(),
        ... 
      }
    )
```